src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
> I was talking about a configuration inside grumphp named intersect_on_commit which will use the intersect path mode during a commit inside GrumPHP. There is no need for any changes in php-cs-fixer for that.

Cool it's gonna be internal setting.
yet, when `intersect_on_commit` would be set up, then you will end up with current issue as well, key is configured, intersection is tried to be used, yet no local config finder is provided.

> Or are you talking about some other way?

if you don't want to have that as dependency and treat it as a blackbox, then:
> poor, quick and dirty solution would be to execute command with intersection mode, if it crashes on not fully defined finder, run it with override mode...